src/screens/ExchangeModal.js

```diff
@@ -500,6 +506,40 @@ export default function ExchangeModal({
     ]
   );
 
+  const navigateToSwapSettingsSheet = useCallback(() => {
+    android && Keyboard.dismiss();
+    const lastFocusedInputHandleTemporary = lastFocusedInputHandle.current;
+    android && (lastFocusedInputHandle.current = null);
+    inputFieldRef?.current?.blur();
+    outputFieldRef?.current?.blur();
+    nativeFieldRef?.current?.blur();
```

Comment:
Isn't blurring one enough?

Reply:
prolly but im just following the other instances